content/posts/0.27-release.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Pyodide 0.27 Release"
+date: 2024-09-27T11:00:55-04:00
+draft: true
+tags: ["announcement"]
+# author: "Me"
+author: ["Gyeongjae Choi"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    # image: "<image path/url>" # image path/url
+    # alt: "<alt text>" # alt text
+    # caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+Pyodide v0.27.0 is out. This release focuses on improving the long-term stability of Pyodide.
+
+# Welcome Agriya Khetarpal to the Pyodide team
+
+Agriya Khetarpal has joined Pyodide as the new maintainer.
+Agriya has been active in the Scientific Python area and is a contributor of Numpy, Scipy, and scikit-learn.
+He has already contributed significantly to strengthening Pyodide support in various Scientific Python packages.
+
+# Pyodide 0.27 Release
+
+## Build System Improvements
+
+### Decoupling pyodide-build from Pyodide runtime
```

Comment:
Maybe we could add the relevant links (PyPI, GitHub, and more) for `pyodide-build` so that users can access them from here?

Reply:
Sounds good.